spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/util/DockerComposeFactory.java

```diff
@@ -169,6 +169,12 @@ public static Extension startDockerCompose(Path tempFolder) {
 	}
 
 	public static Path createTempDirectory() {
+
+		// Temp directory is not required when the docker-compose is not used.
+		if (DockerComposeFactoryProperties.isDockerComposeDisabled()) {
```

Comment:
I wonder why we should call this method DockerComposeFactory when docker compose is not used?

Reply:
It had to be created externally we it can be removed in the @AfterAll. But i will try to replace it whit juni5 statement like this:
```java
@TempDir static Path tempYamlFolder;
```